{"trial_id": "05", "hyperparameters": {"space": [{"class_name": "Int", "config": {"name": "output_units", "default": null, "conditions": [], "min_value": 1, "max_value": 20, "step": 1, "sampling": "linear"}}, {"class_name": "Int", "config": {"name": "layers", "default": null, "conditions": [], "min_value": 2, "max_value": 6, "step": 1, "sampling": "linear"}}, {"class_name": "Float", "config": {"name": "learning_rate", "default": 0.0001, "conditions": [], "min_value": 0.0001, "max_value": 0.1, "step": null, "sampling": "log"}}, {"class_name": "Int", "config": {"name": "units_0", "default": null, "conditions": [], "min_value": 16, "max_value": 256, "step": 16, "sampling": "linear"}}, {"class_name": "Int", "config": {"name": "units_1", "default": null, "conditions": [], "min_value": 16, "max_value": 256, "step": 16, "sampling": "linear"}}, {"class_name": "Int", "config": {"name": "num_layers", "default": 5, "conditions": [], "min_value": 2, "max_value": 50, "step": 1, "sampling": "linear"}}, {"class_name": "Int", "config": {"name": "units", "default": null, "conditions": [], "min_value": 8, "max_value": 64, "step": 8, "sampling": "linear"}}, {"class_name": "Boolean", "config": {"name": "dropout", "default": false, "conditions": []}}], "values": {"output_units": 17, "layers": 5, "learning_rate": 0.006635300540810997, "units_0": 80, "units_1": 144, "num_layers": 29, "units": 64, "dropout": true}}, "metrics": {"metrics": {}}, "score": null, "best_step": 0, "status": "FAILED", "message": "Traceback (most recent call last):\n  File \"c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\keras_tuner\\src\\engine\\base_tuner.py\", line 273, in _try_run_and_update_trial\n    self._run_and_update_trial(trial, *fit_args, **fit_kwargs)\n  File \"c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\keras_tuner\\src\\engine\\base_tuner.py\", line 238, in _run_and_update_trial\n    results = self.run_trial(trial, *fit_args, **fit_kwargs)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\keras_tuner\\src\\engine\\tuner.py\", line 314, in run_trial\n    obj_value = self._build_and_fit_model(trial, *args, **copied_kwargs)\n                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\keras_tuner\\src\\engine\\tuner.py\", line 233, in _build_and_fit_model\n    results = self.hypermodel.fit(hp, model, *args, **kwargs)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\keras_tuner\\src\\engine\\hypermodel.py\", line 149, in fit\n    return model.fit(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\keras\\src\\utils\\traceback_utils.py\", line 70, in error_handler\n    raise e.with_traceback(filtered_tb) from None\n  File \"c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\tensorflow\\python\\framework\\ops.py\", line 1548, in get_attr\n    except errors.InvalidArgumentError as e:\nRuntimeError: pybind11::error_already_set: MISMATCH of original and normalized active exception types: ORIGINAL InvalidArgumentError REPLACED BY KeyboardInterrupt: <EMPTY MESSAGE>\n\nAt:\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\tensorflow\\python\\framework\\errors_impl.py(286): __init__\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\tensorflow\\python\\framework\\ops.py(1546): get_attr\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\tensorflow\\python\\framework\\auto_control_deps_utils.py(105): get_read_write_resource_inputs\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\tensorflow\\python\\framework\\auto_control_deps.py(608): _get_resource_inputs\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\tensorflow\\python\\framework\\auto_control_deps.py(459): __exit__\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\tensorflow\\python\\framework\\func_graph.py(987): func_graph_from_py_func\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\tensorflow\\python\\eager\\polymorphic_function\\tracing_compilation.py(310): _create_concrete_function\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\tensorflow\\python\\eager\\polymorphic_function\\tracing_compilation.py(283): _maybe_define_function\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\tensorflow\\python\\eager\\polymorphic_function\\tracing_compilation.py(178): trace_function\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\tensorflow\\python\\eager\\polymorphic_function\\polymorphic_function.py(695): _initialize\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\tensorflow\\python\\eager\\polymorphic_function\\polymorphic_function.py(888): _call\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\tensorflow\\python\\eager\\polymorphic_function\\polymorphic_function.py(832): __call__\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\tensorflow\\python\\util\\traceback_utils.py(150): error_handler\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\keras\\src\\engine\\training.py(1807): fit\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\keras\\src\\utils\\traceback_utils.py(65): error_handler\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\keras_tuner\\src\\engine\\hypermodel.py(149): fit\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\keras_tuner\\src\\engine\\tuner.py(233): _build_and_fit_model\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\keras_tuner\\src\\engine\\tuner.py(314): run_trial\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\keras_tuner\\src\\engine\\base_tuner.py(238): _run_and_update_trial\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\keras_tuner\\src\\engine\\base_tuner.py(273): _try_run_and_update_trial\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\keras_tuner\\src\\engine\\base_tuner.py(233): search\n  C:\\Users\\hanna\\AppData\\Local\\Temp\\ipykernel_13612\\1713526456.py(3): <module>\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\IPython\\core\\interactiveshell.py(3553): run_code\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\IPython\\core\\interactiveshell.py(3493): run_ast_nodes\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\IPython\\core\\interactiveshell.py(3311): run_cell_async\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\IPython\\core\\async_helpers.py(129): _pseudo_sync_runner\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\IPython\\core\\interactiveshell.py(3106): _run_cell\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\IPython\\core\\interactiveshell.py(3051): run_cell\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\ipykernel\\zmqshell.py(549): run_cell\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\ipykernel\\ipkernel.py(446): do_execute\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\ipykernel\\kernelbase.py(775): execute_request\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\ipykernel\\ipkernel.py(359): execute_request\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\ipykernel\\kernelbase.py(437): dispatch_shell\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\ipykernel\\kernelbase.py(531): process_one\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\ipykernel\\kernelbase.py(542): dispatch_queue\n  C:\\Users\\hanna\\miniconda3\\Lib\\asyncio\\events.py(80): _run\n  C:\\Users\\hanna\\miniconda3\\Lib\\asyncio\\base_events.py(1922): _run_once\n  C:\\Users\\hanna\\miniconda3\\Lib\\asyncio\\base_events.py(607): run_forever\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\tornado\\platform\\asyncio.py(205): start\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\ipykernel\\kernelapp.py(739): start\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\traitlets\\config\\application.py(1075): launch_instance\n  c:\\Users\\hanna\\Documents\\University\\Year 4\\DAML\\env\\Lib\\site-packages\\ipykernel_launcher.py(18): <module>\n  <frozen runpy>(88): _run_code\n  <frozen runpy>(198): _run_module_as_main\n\n"}